CirrusMDSDK.podspec

```diff
@@ -1,7 +1,7 @@
 
 Pod::Spec.new do |s|
   s.name                    = 'CirrusMDSDK'
-  s.version                 = '1.0.2'
+  s.version                 = '1.3.0'
```

Comment:
I'm not sure if it matters but I would change line 16 to 'SWIFT_VERSION' => '4.2'.